packages/host/app/resources/card-resource.ts

```diff
@@ -47,12 +48,13 @@ export interface CardError {
   }[];
 }
 
+export type CardError = CardErrors['errors'][0];
+
 interface Args {
   named: {
     // using string type here so that URL's that have the same href but are
     // different instances don't result in re-running the resource
     url: string | undefined;
-    loader: Loader;
```

Comment:
The loader no longer has to be specified for a CardResource--there only ever is one loader.